research-software-directory/RSD-as-a-service
# fix: deduplicate api responses on ror_id

# Deduplicate RSD and ROR api responses

Closes #909

Changes proposed in this pull request:
* use ror_id as primary key, rsd_path as secondary option for RSD response during deduplication
* homepage, set top image priority based on `linter suggestion`.

How to test:
* `make start` to build app and generate test data
* login as rsd_admin in order to be able to change software/projects
* edit first software from the list
* in organisation section add eScience organisation
* try to add it again. You will see info message that organisation is already added
* exit this software and edit second software from the list
* in organisation section ALSO add eScience organisation. The search list will show only 1 entry (this entry is from RSD api) and you should be able to add eScience to software. If you try again you will see info notification that organisation is already added.
* select first project from the list
* in the info section add eScience as funding organisation. There should be only one option in the list. 
* in the organisation section add eScience again. There should be only one option in the list. 


PR Checklist:
